waspc/data/Generator/templates/react-app/src/queries/index.js

```diff
@@ -9,8 +9,9 @@ export function useQuery(queryFn, queryFnArgs, options) {
     throw new TypeError('queryFn needs to have queryCacheKey property defined.')
   }
 
+  const queryKey = queryFnArgs !== undefined ? [queryFn.queryCacheKey, queryFnArgs] : [queryFn.queryCacheKey]
```

Comment:
**Note:** The third part of the ternary expression can either be `[queryFn.queryCacheKey]` or simply `queryFn.queryCacheKey` (without an array). I decided to always use an array to keep the value more consistent.